.env.example

```
@@ -71,6 +71,23 @@ KEYRING_MAX_VALIDITY_WINDOW_SEC=86400
 # KEYRING_ALLOWED_CHAIN_IDS=0x534e5f5345504f4c4941,0x534e5f4d41494e
 KEYRING_ALLOWED_CHAIN_IDS=
 
+# Signer profile and provider mode
+KEYRING_SECURITY_PROFILE=flex
+KEYRING_SIGNER_PROVIDER=local
+KEYRING_SIGNER_FALLBACK_PROVIDER=none
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYRING_SIGNER_FALLBACK_PROVIDER key should go before the KEYRING_SIGNER_PROVIDER key

(UnorderedKey)